app/views/volunteers/add_repairs.html.erb

```diff
@@ -0,0 +1,40 @@
+<% @repairs.each do |repair| %>
+<% if VolunteerRepair.where('volunteer_id = ? AND repair_id = ?', @volunteer.id, repair.id).length == 0 %> 
+    Repair Type: <%= repair.repair_type %><br/>
+    Repair Notes: <%= repair.repair_notes %><br/>
+    <div>
+    <%= link_to 'Sign up for a repair', add_repair_to_volunteer_volunteer_path(repair.id), method: :post %>`
+    </div>
+    <% end %>
+<% end %>
+<br/>
+<h2>
+<p>Everything Below I've sign up for</p>
+</h2>
+
+<% @volunteer.volunteer_repairs.each do |volunteer_repair| %>
+     Repair Type: <%= volunteer_repair.repair.repair_type %><br/>
+    Repair Notes: <%= volunteer_repair.repair.repair_notes %><br/>
+    <div>
+    <%= link_to 'Remove', remove_repair_volunteer_path(volunteer_repair.id), method: :delete %>`|
+    <%= link_to 'Submit', submit_volunteer_path(volunteer_repair.id), method: :put %>`
+    </div>
+<% end %>
+
```

Comment:
Here I don't know why we need a submit process or a submitted section.  I think the volunteer_repairs should be created as "Pending" and the step should be eliminated.  What am I missing?